Gajeongtongsin/Gajeongtongsin/Screens/Teacher/Consultation/ConsultationViewController.swift

```diff
@@ -78,15 +84,29 @@ class ConsultationViewController: BaseViewController {
         super.viewDidLoad()
         calenderView.delegate = self
         calenderView.dataSource = self
+        customNavigationBar.delegate = self
```

Comment:
3개이상 부터는 가독성을 위해서
```suggestion
func setDelegations() {
        calenderView.delegate = self
        calenderView.dataSource = self
        customNavigationBar.delegate = self
        }
```
이렇게 만들어서 함수 호출하면 어떨까요?

Reply:
반영했습니다